website/notifications/utils.py

```diff
@@ -176,8 +176,11 @@ def get_configured_projects(user):
         # If the user has opted out of emails skip
         node = subscription.owner
 
-        if not isinstance(node, Node) or (user in subscription.none and not node.parent_id) or node.is_bookmark_collection:
-            continue
+        if (not isinstance(node, Node) or
+                (user in subscription.none and not node.parent_id) or not
+                node.notification_settings_dirty or
+                node.is_bookmark_collection):
```

Comment:
Minor: I realize this was here before, but I think we should skip over any collection (not just Bookmarks), i.e. `node.is_collection`.